From: Martin Pitt <mpitt@debian.org>
Date: Mon, 14 Nov 2016 22:41:23 +0100
Subject: Disable plugin dialog

It uses a totally non-authenticated and non-trusted way of installing arbitrary
code.

Bug-Debian: http://bugs.debian.org/640026
---
 src/calibre/gui2/actions/preferences.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/calibre/gui2/actions/preferences.py b/src/calibre/gui2/actions/preferences.py
index 65edb4d..50183da 100644
--- a/src/calibre/gui2/actions/preferences.py
+++ b/src/calibre/gui2/actions/preferences.py
@@ -28,8 +28,6 @@ class PreferencesAction(InterfaceAction):
             pm.addAction(QIcon(I('config.png')), _('Preferences'), self.do_config)
         cm('welcome wizard', _('Run welcome wizard'),
                 icon='wizard.png', triggered=self.gui.run_wizard)
-        cm('plugin updater', _('Get plugins to enhance calibre'),
-                icon='plugins/plugin_updater.png', triggered=self.get_plugins)
         if not DEBUG:
             pm.addSeparator()
             cm('restart', _('Restart in debug mode'), icon='debug.png',
